src/app/mod.rs

```
@@ -425,9 +427,10 @@ impl Surreal {
 			Ok(range) => self.db.update(Resource::from(range.tb)).range((range.beg, range.end)),
 			Err(_) => self.db.update(Resource::from(resource)),
 		};
-		let response = match from_value::<Option<Json>>(data)? {
-			Some(data) => update.content(data).await?,
-			None => update.await?,
+		let data = json(&from_value::<Json>(data)?.to_string()).unwrap_or(Value::None);
```

Same here. Can we please return the error, not fall back to `Value::None`?